src/Commands/Disable.php

```diff
@@ -11,22 +11,16 @@ class Disable extends TaskCommand
 {
     /**
      * The Command's name
-     *
-     * @var string
```

Comment:
Why is `@var` removed?

Reply:
My mistake, see comment below.

Reply:
Actually, these are all typed in the parent class so don't need a `@var` tag here. Probably safe to remove the comment altogether.